xml/sec_cf_usb_url.xml

```diff
@@ -72,7 +72,7 @@
     Get the current <literal>cf-usb</literal> password. Use the name of the
     secret obtained in the first step:
    </para>
-<screen>&prompt.user;kubectl get secret --namespace scf <replaceable>secrets-2.15.1-1</replaceable> -o yaml | grep \\scf-usb-password: | cut -d: -f2 | base64 -id</screen>
+<screen>&prompt.user;kubectl get secret --namespace scf <replaceable>secrets-2.15.2-1</replaceable> -o yaml | grep \\scf-usb-password: | cut -d: -f2 | base64 -id</screen>
```

Comment:
Does it make sense to use `&latestscfchart;` here, so we can replace less in the future?

Reply:
:heavy_check_mark:

Reply:
:heavy_check_mark: